simplq/src/store/appSlice.js

```diff
@@ -68,6 +69,10 @@ const appSlice = createSlice({
       // eslint-disable-next-line no-param-reassign
       state.errorText = action.payload;
     },
+    setNotificationPermission: (state, action) => {
+      // eslint-disable-next-line no-param-reassign
+      state.notificationPermission = action.payload;
```

Comment:
lets also store to localstorage?

window.localStorage.setItem('notificationPermission', action.payload);

Reply:
Why do we need to do this?

Reply:
Only if you see the need to. My thought was, in the old chrome browser scenario that you were telling before, `Notification.permission` will be `undefined`.

So the first time user visits, we will ask for permission and set the user response to `Notification.permission`:

```
      // Some older versions of chrome doesn't store the permission in the Notification object. In that case we store it manually.
      if (!('permission' in Notification)) {
        Notification.permission = permission;
      }
```

I was wondering if the user will again be asked for notification next time he visits, after reopening the browser, if it's not persisted to localstorage.